miden-lib/asm/miden/kernels/tx/constants.masm

```diff
@@ -4,20 +4,20 @@
 # The number of elements in a Word
 const.WORD_SIZE=4
 
-# The maximum number of assets that can be stored in a single note
-const.MAX_ASSETS_PER_NOTE=255
+# The maximum number of assets that can be stored in a single note.
+const.MAX_ASSETS_PER_NOTE=256
```

Comment:
I believe we used `255` in the past so that it could be encoded in a single byte. However, since a note with 0 assets is not allowed, we can encode `256` into a single byte as well (we just serialize it as `num_assets - 1`). Rust code was able to handle this for a while, but we forgot to update the MASM code.